web_responsive/readme/DESCRIPTION.md

```diff
@@ -2,6 +2,10 @@ This module adds responsiveness to web backend.
 
 **Features for all devices**:
 
+- Redirect to the dashboard after logging in
+
+  ![image](../static/img/redirecthome.gif)
+
```

Comment:
Redirect to the home page after login will occur only if the user has enabled the 'Redirect to Home' configuration in their user profile settings

Reply:
@francesco-ooops 
I've updated it according to the suggestion.